docs/articles/documentation/test-api/README.md

```diff
@@ -16,6 +16,7 @@ It contains the following subsections
 * [Assertions](assertions/README.md)
 * [Obtaining Data From the Client](obtaining-data-from-the-client.md)
 * [Waiting for Page Elements to Appear](waiting-for-page-elements-to-appear.md)
+* [User Roles](user-roles.md)
```

Comment:
It better be put in section named `Authentication` and move `HTTP Authentication` to that section as well